web_app/dist/main.css

```diff
@@ -0,0 +1,151 @@
+body {
+    font-family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+    font-size: 18px;
+    height: 100%;
+    background: #93ca68;
+    background-image: url(images/bg_logo.png);
+    background-repeat: repeat;   
+}   
+
+#content {
+    background: #fff;
+    color: #545454;
+    line-height: 1.5;
+    margin: 0 auto;
+    max-width: 60%;
+    height: 90%;
+    margin-top: 50px;
+    padding-bottom: 20px;
+    border-radius: 10px;
+    box-shadow: 5px 25px 20px -10px #606060; 
+}
+
+#sticky-header {
+    display: flex;
+    flex-direction: column;
+    position: fixed;
+    top: 0;
+    left: 0;
+    width: 100%;
+}
+
+.navbar {
+    display: flex;
+    overflow: hidden;
+    background-color: #333;
+    width: 100%;
+    box-shadow: -5px 20px 50px -10px #656565; 
+}
+
+#search-bar {
+    padding: 2px;
+    border: none;
+    margin-top: 10px;
+    margin-right: 16px;
+    margin-left: auto;
+    margin-bottom: 10px;
+    font-size: 17px;
+    border-radius: 5px;
+}
+
+.navbar a {
+    color: #f2f2f2;
+    text-align: center;
+    padding: 14px 16px;
+    text-decoration: none;
+}
+
+.navbar a:hover {
+    background-color: #ddd;
+    color: black;
+}
+
+.task {
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+    background-color:honeydew;
```

Comment:
Add space after `:`, make sure this doesn't happen elsewhere

Reply:
kk